layers/meta-balena-compulab/recipes-kernel/linux/linux-imx_4.14.%.bbappend

```diff
@@ -19,6 +19,7 @@ SRC_URI_append_etcher-pro = " \
 	file://0001-Add-support-for-KSZ9893R.patch \
 	file://0001-Enable-KSZ9893R-switch-in-Etcher-Pro-dtb.patch \
 	file://0007-HDMI-Set-no-EDID.patch \
+	file://0008-Add_ep_config.patch \
```

Comment:
You should mention in the git log where this config is taken from.
And if you want to add this defconfig, you can just add it as a file in the yocto layer and configure the kernel to use it. As this patch adds it now, how is the build system instructed to use this defconfig?
Also, as a NIT there is a typo in the git log: `requried`

Reply:
I think it's due to the BSP update. The defconfig is linked to the machine name. When I triggered the etcher-pro build, it failed, asking for etcher-pro_defconfig

Reply:
done